### PR TITLE
Add a list of plugins and note how they are used

### PR DESCRIPTION
**Pull request by rgildein**
Unfortunately it is not possible to enable the use of custom plugins, since it could break the tempest itself. Some plugins has tempest as requirement.

### CODE REVIEW

**Review by esunar — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -8,3 +8,43 @@ test suite, Tempest.
 The tempest snap can be installed directly from the snap store:
 
     sudo snap install --edge tempest
+
+
+## Plugins
+
+The image is baked with all plugins listed in the release notes and those cannot be
```
Comment (agileshaw) [inline README.md:15]: nit: it would be better to provide a link to the release notes you mentioned.

**Review by agileshaw — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
 ## This snap includes
 
 ### python-tempestconf
 
-This snapshot includes [python-tempestconf](https://opendev.org/openinfra/python-tempestconf),
-which automatically generates a tempest configuration based on your cloud.
+This snapshot includes [python-tempestconf], which automatically generates a tempest
+configuration based on your cloud.
+
+It provided single CLI options [`discover-tempest-config`] which is provided in snap
```
Comment (agileshaw) [inline README.md:77]:
```suggestion
It provided a single CLI option [`discover-tempest-config`] which is provided in snap
```

File: README.md [hunk 3]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
+- venus-tempest-plugin
+- vitrage-tempest-plugin
+- watcher-tempest-plugin
+- zaqar-tempest-plugin
+- zun-tempest-plugin
+
 
 ## Quickstart
-To run Tempest, please follow the steps in [Tempest QuickStart](https://docs.openstack.org/tempest/latest/overview.html#quickstart) documentation.
+To run Tempest, please follow the steps in [Tempest QuickStart] documentation.
```
Comment (aieri) [inline README.md:54]:
```suggestion
To run Tempest, please follow the steps in the [Tempest QuickStart] documentation.
```

File: README.md [hunk 4]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
 
-The tempest snap provides a set of pre-defined test lists with different focuses. For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
+The tempest snap provides a set of pre-defined test lists with different focuses.
+For quickly verifying the behavior of the objective cloud and making sure tempest
```
Comment (aieri) [inline README.md:57]:
```suggestion
For quickly verifying the behavior of the target cloud and making sure tempest
```

File: README.md [hunk 5]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
+is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
 
     tempest run --load-list '@BUILTIN_TESTLISTS/readonly-quick.txt' 
 
-`@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of the directory storing the built-in test lists.
+`@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of
```
Comment (aieri) [inline README.md:62]:
```suggestion
The `@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of
```

File: README.md [hunk 6]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
+the directory storing the built-in test lists.
 
-**Note**: the single-quotation marks around the test list file path is required if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest with a built-in test list file). Fail to do that will result in wrong path error.
+**Note**: the single-quotation marks around the test list file path is required
+if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest
```
Comment (aieri) [inline README.md:66]:
```suggestion
If the `@BUILTIN_TESTLISTS` keyword is used (in other words, when running tempest
```

File: README.md [hunk 7]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
+with a built-in test list file). Fail to do that will result in wrong path error.
```
Comment (aieri) [inline README.md:67]:
```suggestion
with a built-in test list file), failure to do so will result in a wrong path error.
```

File: README.md [hunk 8]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
 
 
 ## This snap includes
 
 ### python-tempestconf
 
-This snapshot includes [python-tempestconf](https://opendev.org/openinfra/python-tempestconf),
-which automatically generates a tempest configuration based on your cloud.
+This snapshot includes [python-tempestconf], which automatically generates a tempest
```
Comment (aieri) [inline README.md:74]:
```suggestion
This snap includes [python-tempestconf], which automatically generates a tempest
```

File: README.md [hunk 9]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
+configuration based on your cloud.
```
Comment (aieri) [inline README.md:75]:
```suggestion
configuration appropriate for your cloud.
```

File: README.md (hunk 2 above)
Comment (aieri) [inline README.md:77]:
```suggestion
It provides a single CLI command [`discover-tempest-config`] which is aliased in the snap
```

File: README.md [hunk 10]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
+
+It provided single CLI options [`discover-tempest-config`] which is provided in snap
+via `tempest.discover`.
```
Comment (aieri) [inline README.md:78]:
```suggestion
as `tempest.discover`.
```

File: README.md [hunk 11]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
...
 
-It provided single CLI options [`discover-tempest-config`](https://docs.opendev.org/openinfra/python-tempestconf/latest/cli/cli_options.html#discover-tempest-config),
-which is provided in snap via `tempest.discover`.
+[OpenStack release notes]: https://releases.openstack.org/antelope/index.html#tempest-plugins
```
Comment (aieri) [inline README.md:80]: too bad there's no `https://releases.openstack.org/latest/index.html#tempest-plugins`...

File: README.md [hunk 12]
```diff
@@ -9,25 +9,75 @@ The tempest snap can be installed directly from the snap store:
 
     sudo snap install --edge tempest
```
[the remark this answers is not in the document]
Comment (aieri) [reply README.md:10]: This README goes into the master branch so I'm thinking it'd be better to be more generic and don't assume `--edge`

**Review by aieri — CHANGES_REQUESTED** [review]
A few rewording suggestions

File: README.md (hunk 11 above)
Comment (agileshaw) [reply README.md:80]: Yes, unfortunately we have to update this each time a new os release is out.

File: README.md [hunk 13]
```diff
@@ -7,27 +7,77 @@ test suite, Tempest.
...
+- zaqar-tempest-plugin
+- zun-tempest-plugin
 
 
 ## Quickstart
-To run Tempest, please follow the steps in [Tempest QuickStart](https://docs.openstack.org/tempest/latest/overview.html#quickstart) documentation.
+To run Tempest, please follow the steps in the [Tempest QuickStart] documentation.
 
-The tempest snap provides a set of pre-defined test lists with different focuses. For quickly verifying the behavior of the objective cloud and making sure tempest is correctly set up, you can run tests listed in the `readonly-quick.txt` file:
+The tempest snap provides a set of pre-defined test lists with different focuses.
```
Comment (aieri) [inline README.md:56]: well, only one for now...

File: README.md [hunk 14]
```diff
@@ -7,27 +7,77 @@ test suite, Tempest.
...
 
     tempest run --load-list '@BUILTIN_TESTLISTS/readonly-quick.txt' 
 
-`@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path of the directory storing the built-in test lists.
+The `@BUILTIN_TESTLISTS` keyword will be automatically replaced by the full path
+of the directory storing the built-in test lists.
 
-**Note**: the single-quotation marks around the test list file path is required if `@BUILTIN_TESTLISTS` keyword is used (in another words, when running tempest with a built-in test list file). Fail to do that will result in wrong path error.
+**Note**: the single-quotation marks around the test list file path is required
+if the `@BUILTIN_TESTLISTS` keyword is used (in other words, when running tempest
```
Comment (aieri) [inline README.md:66]:
```suggestion
If the `@BUILTIN_TESTLISTS` keyword is used (in other words, when running tempest
```

**Review by aieri — COMMENTED** [review]
LGTM, just one tiny incorrect capitalization

**Review by aieri — APPROVED** [review]

File: README.md (hunk 14 above)
Comment (agileshaw) [reply README.md:66]: As discussed elsewhere, `if` should be kept lower-cased.

File: README.md (hunk 13 above)
Comment (agileshaw) [reply README.md:56]: As discussed elsewhere, we are keeping this wording plural because more test lists are going to be added in the following PRs.

**Review by agileshaw — APPROVED** [review]

**Review by gabrielcocenza — APPROVED** [review]
LGTM

**Pull request merged**